fhir-audit/src/main/java/com/ibm/fhir/audit/cadf/model/CadfEvent.java

```diff
@@ -765,50 +765,64 @@ public static void generate(CadfEvent obj, JsonGenerator generator)
             if (obj.getTags() != null) {
                 generator.writeStartArray("tags");
                 for (String tag : obj.getTags()) {
-                    generator.write(tag);
+                    if (tag != null) {
```

Comment:
Can the CadfEvent (obj) really contain nulls in its repeating members?

Reply:
obj.getTags is a list, so technically it can.